calSprite.py

```diff
@@ -312,12 +312,18 @@ async def notice(self, text):
         msg = text[text.find(parameters[1][1:]) :]
 
         if msg.upper().startswith("REPORT"):
-            currTime = datetime.datetime.now
-            await self.send(f"PRIVMSG #reports {nick}: {msg} - at ",  )
-            
-            await self.send(f"PRIVMSG {nick} Report send :3 (mods can only see reports"
-                            " if they're online, might be better to ping "
-                            "a mod on discord if it's urgent)")
+            # tested these two lines below on IDLE, seemed to work well
+            inboundReport = msg.split(' ', 2)
+            reportMsgUser, reportMsgBody = inboundReport[1:]
+
+            # NOTE: untested since i didn't have the time to setup a local irc server (rip scratchware...)
```

Comment:
shou please remind me to set up an unrealircd instance locally next time i talk about proposing changes to calsprite